src/main/webapp/scripts/keywords.js

```diff
@@ -1,6 +1,47 @@
 const BLOB_URL_PROMISE = loadTemplate('/user-image');
 
 $().ready(function() {
+  fetch('/login').then(response => response.json()).then((logURL) => {
```

Comment:
[Optional] Instead of fetch is load.js functions to load object and then use await instead of .then. You would also have to mark the callback function as async.

Reply:
The same way it is being done for '/user-image' and the BLOB_URL_PROMISE.